src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx

```diff
@@ -70,12 +72,37 @@ const Row = ({ children, Icon, label }) => {
  * @component
  */
 const MissionDetailsCard = ({ mission, photoDisabled }) => {
-  photoDisabled = true;
+  const snackbarContext = useContext(Snackbar.Context.SnackbarContext);
+  const firebase = getFirebase();
```

Comment:
Should use `useFirebase` here.

`getFirebase` isn't a hook and therefore can't update react fire something changes